app/js/app.js

```diff
@@ -35,6 +35,57 @@ window.addEventListener('DOMContentLoaded', () => {
+  font-weight: 500;
+  font-style: normal;
+}
+
+[data-icon]:before,
+.ligature-icons {
+  font-family: "gaia-icons";
+  content: attr(data-icon);
+  display: inline-block;
+  font-weight: 500;
+  font-style: normal;
+  text-decoration: inherit;
+  text-transform: none;
+  text-rendering: optimizeLegibility;
+  font-size: 30px;
+  -webkit-font-smoothing: antialiased;
```

Comment:
This is generated by a grunt task somewhere for generating the font icon stylesheet.